src/Utilities/Compiler/Extensions/CompilationExtensions.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#pragma warning disable IDE0005 // Using directive is unnecessary - used in conditional compilation.
```

Comment:
Instead of applying this pragma to the entire set of usings, can you instead narrow it to only the usings used in conditional compilation?